_sass/custom/setup.scss

```diff
@@ -0,0 +1,6 @@
+// custom setup code goes here
+
+$pink-000: #f77ef1;
+$pink-100: #f967f1;
+$pink-200: #e94ee1;
+$pink-300: #dd2cd4;
```

Comment:
I think we should avoid content in such a custom file, because changing the content could _break_ sites that rely on the content but _don't_ override the file.

So I suggest to delete these lines.

Reply:
Hm, ok - in that case, this theme will introduce different behaviour (since we're removing these same lines from `custom.scss`.

Reply:
I think those lines were added in RC1, by #578. So it's not breaking wrt v0.3.3.

And yes, this illustrates very well the danger of putting content in custom styles…